### PR TITLE
updated wiki.json

### PR DESCRIPTION
**Pull request by comrade-om**
## Description
updated wiki.json file as per the specified instructions

## Changes
added key value pairs where the key name is same as the first argument passed to the constructor in SlimefunItemStack and the value name is same as the name of the wiki it is referring to.

## Related Issues
Resolves [#2389](https://github.com/Slimefun/Slimefun4/issues/2389)

## Checklist
- [ ] I have fully tested the proposed changes and promise that they will not break everything into chaos.
- [ ] I have also tested the proposed changes in combination with various popular addons and can confirm my changes do not break them.
- [x] I followed the existing code standards and didn't mess up the formatting.
- [ ] I did my best to add documentation to any public classes or methods I added.
- [ ] I have added `Nonnull` and `Nullable` annotations to my methods to indicate their behaviour for null values
- [ ] I added sufficient Unit Tests to cover my code.

### CODE REVIEW

File: src/main/resources/wiki.json [hunk 1]
```diff
@@ -99,5 +99,149 @@
...
+	"GLOWSTONE_LEGGINGS" : "Armor",
+	"GLOWSTONE_BOOTS" : "Armor",
+	"ENDER_HELMET" : "Armor",
+	"ENDER_CHESTPLATE" : "Armor",
+	"ENDER_LEGGINGS" : "Armor",
+	"ENDER_BOOTS" : "Armor",
+	"SLIME_HELMET" : "Armor",
+	"SLIME_CHESTPLATE" : "Armor",
+	"SLIME_LEGGINGS" : "Armor",
+	"SLIME_BOOTS" : "Armor",
```
Comment (LinoxGH) [inline src/main/resources/wiki.json:121]: These need to be "Magical-Armor"

File: src/main/resources/wiki.json [hunk 2]
```diff
@@ -99,5 +99,149 @@
...
+	"GILDED_IRON_LEGGINGS" : "Armor",
+	"GILDED_IRON_BOOTS" : "Armor",
+	"GOLD_12K_HELMET" : "Armor",
+	"GOLD_12K_CHESTPLATE" : "Armor",
+	"GOLD_12K_LEGGINGS" : "Armor",
+	"GOLD_12K_BOOTS" : "Armor",
+	"SLIME_STEEL_HELMET" : "Armor",
+	"SLIME_STEEL_CHESTPLATE" : "Armor",
+	"SLIME_STEEL_LEGGINGS" : "Armor",
+	"SLIME_STEEL_BOOTS" : "Armor",
```
Comment (LinoxGH) [inline src/main/resources/wiki.json:145]: Should be in "Magical-Armor"

File: src/main/resources/wiki.json [hunk 3]
```diff
@@ -99,5 +99,149 @@
 	"ORE_CRUSHER" : "Ore-Crusher",
 	"PRESSURE_CHAMBER" : "Pressure-Chamber",
 	"GRIND_STONE" : "Grind-Stone",
-	"MAGIC_WORKBENCH" : "Magic-Workbench"
+	"MAGIC_WORKBENCH" : "Magic-Workbench",
+	"PORTABLE_CRAFTER" : "Portable-Crafter",
+	"PORTABLE_DUSTBIN" : "Portable-Dustbin",
+	"COOLER" : "Cooler",
+	"ALUMINUM_BRASS_INGOT" : "Aluminium-Brass-Ingot",
```
Comment (LinoxGH) [inline src/main/resources/wiki.json:106]: Aluminum*

File: src/main/resources/wiki.json [hunk 4]
```diff
@@ -99,5 +99,149 @@
...
+	"RAINBOW_GLASS" : "Rainbow-Blocks",
+	"RAINBOW_CLAY" : "Rainbow-Blocks",
+	"RAINBOW_GLASS_PANE" : "Rainbow-Blocks",
+	"RAINBOW_CONCRETE" : "Rainbow-Blocks",
+	"RAINBOW_GLAZED_TERRACOTTA" : "Rainbow-Blocks",
+	"REDSTONE_ALLOY" : "Redstone-Alloy-Ingot",
+	"REINFORCED_ALLOY_INGOT" : "Reinforced-Alloy-Ingot",
+	"SILVER_DUST" : "Silver-Dust",
+	"SILVER_INGOT" : "Silver-Ingot",
+	"SOLDER_INGOTNICKEL_INGOT" : "Solder-Ingot",
```
Comment (LinoxGH) [inline src/main/resources/wiki.json:224]: ...

**Review by LinoxGH — CHANGES_REQUESTED** [review]

**Comment by comrade-om** [comment]
@LinoxGH yes on it, will do the needful changes. Everything else alright?

File: src/main/resources/wiki.json (hunk 4 above)
Comment (comrade-om) [reply src/main/resources/wiki.json]: done :smile:

File: src/main/resources/wiki.json (hunk 3 above)
Comment (comrade-om) [reply src/main/resources/wiki.json]: done :smile:

File: src/main/resources/wiki.json (hunk 1 above)
Comment (comrade-om) [reply src/main/resources/wiki.json]: done :smile:

File: src/main/resources/wiki.json (hunk 2 above)
Comment (comrade-om) [reply src/main/resources/wiki.json]: done :smile:

**Review by TheBusyBiscuit — APPROVED** [review]
I couldn't find any more mistakes, but @LinoxGH and @svr333 will still need to review this before it will be merged.

Thank you for making this pull request <3

**Comment by comrade-om** [comment]
@TheBusyBiscuit thank you very much for reviewing my pull request. Can this get the `hacktoberfest-accepted` label so that the 14day timer will start as per the updated rules :pray: :heart:

**Review by svr333 — APPROVED** [review]
Didnt see any mistakes

**Comment by TheBusyBiscuit** [comment]
> @TheBusyBiscuit thank you very much for reviewing my pull request. Can this get the `hacktoberfest-accepted` label so that the 14day timer will start as per the updated rules 🙏 ❤️

Sorry that I didn't see this message earlier.
Pull Requests for Hacktoberfest already start counting once they have been approved, there is no need for the label.
And since we got two approving reviews now, I can merge this anyway.
Thanks for making this pull request and I wish you a fun Hacktoberfest 2020 :tada:

**Pull request merged**